src/check/parse/IR.zig

```diff
@@ -1761,6 +1814,75 @@ pub const NodeStore = struct {
         },
 
         const TagUnionRhs = packed struct { open: u1, tags_len: u31 };
+
+        pub fn toSExpr(self: @This(), env: *base.ModuleEnv, ir: *IR) sexpr.Expr {
+            switch (self) {
+                // (ty_var <var>)
+                .ty_var => |a| {
```

Comment:
I really hate that this is `ty_var` and the other is `type_anno`....so inconsistent....but not from this PR or something you need to fix to merge.